src/main/channels/audio/GenerateAudiosTranscriptionChannel.ts

```
@@ -0,0 +1,38 @@
+import { IpcChannel } from '@main/interfaces/IpcChannel';
+import { GENERATE_AUDIO_TRANSCRIPTION_CHANNEL } from '@shared/channels';
+import { GenerateAudioTranscriptionParams } from '@shared/interfaces/Audio';
+import { IpcRequest } from '@shared/interfaces/IpcRequest';
+import { instanceToPlain } from 'class-transformer';
+import { IpcMainEvent } from 'electron';
+import log from 'electron-log/main';
+
+import { AppDataSource } from '../../data-source';
```

ESLint

[error] 'AppDataSource' is defined but never used.

(@typescript-eslint/no-unused-vars)